File: solutions/implementation/parachute.java
import java.util.*;
public class Main
{
	public static void main(String[] args) 
	{
	    Scanner input = new Scanner(System.in);
	    int b = 5, c;
	    double a = -4.9, t;
	    c = 360-input.nextInt();
	    t = (-b-Math.sqrt(b*b-4.0*a*c))/(2.0*a); //calculate time using quadratic formula
		System.out.format("%.1f", t); //use format to format to one decimal place
	}
} 

File: solutions/implementation/parachute.py
import math

# to get how long Mr. Fraschetti took to open his parachute we must solve a system of equations
# by subbing the parabolic equation of his arc before deploying the parachute to the linear one we get -4.9t^2 + 5t + 360-b
b = int(input())

# using the quadratic formula
discriminant = 5**2 - (4*(-4.9)*(360-b))
print(round((-5-math.sqrt(discriminant))/(2*(-4.9)), 1))
